Source/Relativity.Transfer.SDK.Sample/App.config

```diff
@@ -17,10 +17,18 @@
 			 You can modify this value and observe the transfer result in for example RelativityOne Staging Explorer).
 			 The path has to be rooted in one of the core folders that reside on fileshare (like Files, Temp, ARM, etc.) -->
 		<add key="FileshareRelativeDestinationPath" value="Temp\TransferSDK-Sample" />
-
+		
+		<!-- For upload it should be local file path. -->
+		<!-- For download it should be relative path to the `FileshareRelativeDestinationPath`. -->
 		<add key="DefaultSourceFilePath" value=""/>
+		
+		<!-- For upload it should be local directory path. -->
+		<!-- For download it should be relative path to the `FileshareRelativeDestinationPath`. -->
 		<add key="DefaultSourceDirectoryPath" value=""/>
 		
+		<!-- Absolute path to directory where files/directories will be downloaded.-->
+		<add key="DownloadCatalog" value=""/>
```

Comment:
why do not use DestinationPath and for upload it will be FileshareRelativeDestinationPath, but for download it will be the local path